### ISSUE TITLE
[Question] Is it possible to adapt this to other OEMs?

### ISSUE THREAD

**Issue by lxwulf** [lenovo/lenovo-wwan-unlock#14]
# Explanation

This worked flawlessly on my side, `ThinkPad X1 Yoga` | `Fibocom FM350-GL 5G SUB6-Modem`.
I never thought that such new wwan module getting working on Linux. 

So I thought if it's possible to use this script with other devices which has the same wwan module installed but are not from Lenovo.

**Comment by nitinexclusively** [comment]
@lxwulf  Thanks for checking it out. This is only intended for Lenovo machine mentioned in https://github.com/lenovo/lenovo-wwan-unlock/blob/main/README.md

**Comment by lxwulf** [comment]
Yes, I saw this but thought maybe we can add those processes to other OEMs wich has the same `WWAN Module` and :point_up: of course installed one of the mentioned Linux distributions.

Is the script depending on hardware which is only installed on the Lenovo laptops, or why does it not work? I think it would be a very nice addition to Linux in general.

Of course, I respect this here, and I'm happy I chose the right OEM :wink:, so this here is just knowledge question. :blush:
